monitoring/settings.py

```diff
@@ -11,11 +11,12 @@
 """
 
 import os
+import configparser
+import sys
```

Comment:
```suggestion
import configparser
import os
import sys
```
Change imports to alphabetical order.